cmd/tailscale/cli/serve.go

```diff
@@ -725,3 +751,16 @@ func (e *serveEnv) runServeReset(ctx context.Context, args []string) error {
 	sc := new(ipn.ServeConfig)
 	return e.lc.SetServeConfig(ctx, sc)
 }
+
+// parseValidServePort parses a port number from a string and returns it as a
+// uint16. It returns an error if the port number is invalid or zero.
+func parseValidServePort(s string) (uint16, error) {
```

Comment:
I think the word "Valid" is kinda redundant here. It returns an error; that basically already says that it returns an error if the input is not valid. parseServePort works just as well.

Reply:
Removed!